### ISSUE TITLE
[design] Re-branding?

### ISSUE THREAD

**Issue by corysimmons** [mojotech/stylus-redesign#1]
I think we should rebrand a bit. Not much, but I think the logo should be cleaned up with a slightly more luscious font, proper kerning, and a cleaned up `yl` mark.

I don't think we'd need to get @visionmedia approval on this since it's such a small change.

**Comment by kizu** [comment]
Yep, if it would be just cosmetic changes to the existing logo, I don't think it would need an approval.

**Comment by corysimmons** [comment]
@thefoxis Hook us up? If you can't get to it, I can get one this weekend. I'm not familiar with LayerVault other than it's Github for design resources, but I'm sure I can figure it out.

**Comment by thefoxis** [comment]
I have a LayerVault account. I can invite you to the project if you give me your emails :)

**Comment by corysimmons** [comment]
cory@mojotech.com

**Comment by corysimmons** [comment]
@thefoxis I just made an account so you can add me whenever.

**Comment by thefoxis** [comment]
@corysimmons &mdash; added :)

**Comment by kizu** [comment]
kizmarh@ya.ru

**Comment by corysimmons** [comment]
Just added a logo to the vault. It's vector.

![](http://i.imgur.com/OC99VrN.png)

**Comment by corysimmons** [comment]
@kizu We have 2 of our best guys working on fresh brand concepts for this as well: http://dribbble.com/mattrossidesigns http://dribbble.com/adder

**Comment by thefoxis** [comment]
@corysimmons &mdash; I'm happy to give up the branding to Ivan and Matt. I do UIs mostly, I think they will come up with better concepts for the logo than I do :) Should I just add them to LayerVault?

@kizu &mdash; added.

**Comment by corysimmons** [comment]
@thefoxis - Awesome. Yeah, I'm not much of a logo designer myself. I'm really excited to see the UI you come up with! :D

Feel free to add them:
- mrossi@mojotech.com
- ivan@mojotech.com

**Comment by thefoxis** [comment]
Invited. The project is [here](https://layervault.com/karolina-szczur/Stylus) and it's public :)

There are two folders, one for branding and one for the website. I couldn't find Ivan and Matt on Github, but it would be nice if they could save previews in branding/previews. I just do it for consistency sake. 

:confetti_ball:

**Comment by corysimmons** [comment]
@thefoxis I'm not sure how long it usually takes, or why it's happening, but I can't see my `.ai` in the web gui for LayerVault. Am I doing something wrong?

![](http://i.imgur.com/JfzdJIl.png)

Sorry, I didn't realize they needed to be on Github. I'm unsure if either use Github since they're designers.

**Comment by tj** [comment]
the logo/site were just quick hacks, totally cool with someone doing rebranding if they're into it

**Comment by thefoxis** [comment]
@corysimmons we can collaborate on LV. no need for them to be on Github for that. I was just wondering.

`previews` is for _.pngs/_.jpgs and the root (here branding) is for sources. it should show up after a while, if it doesn't try restarting LV (sometimes it freezes). I also changed the permissions from _Observer_ to _Designer_, so should work now :)

@visionmedia &mdash; great! thanks :sunflower:

**Comment by corysimmons** [comment]
:clap:

**Comment by corysimmons** [comment]
@thefoxis You have any sketches for the UI?

**Comment by thefoxis** [comment]
nope, we should figure out the sitemap and content first.

**Comment by corysimmons** [comment]
Oh yeah! :+1: 

Sorry, doing 50 things.

https://github.com/corysimmons/stylus-redesign/issues/4

**Comment by corysimmons** [comment]
Ivan made this one. I think we can use it while we design/develop the UI.

![](http://i.imgur.com/dPIB8uB.png)

**Comment by thefoxis** [comment]
Logo is secondary, we can add it basically whenever. I need to look into fonts and overall color scheme. I think the color scheme should somewhat revolve around green (Node main color), but need to add some variations and accent colors.

Regarding the logo I'd imagine in being somewhat connected to "stylus" as a word, so pen, stylus, etc. Otherwise I'd gladly see it as just a word, but nicely put (in a typographic sense). Because stylus = style = nice, handwritten font. Kind of like Sass logo.

**Comment by corysimmons** [comment]
There's a brush tip.

I think Node-green is a fine idea for an accent color. Without ripping them off directly, I love their new, dark, color scheme: http://nodejs.org

I think it's easiest to read doc blocks with typography and a color-scheme like Github's tbh.

**Comment by corysimmons** [comment]
@thefoxis Wouldn't it be cool if we had a mascot?

It's something the other 2 preprocessors don't have and opens the door to all kinds of illustrations.

**Comment by thefoxis** [comment]
I'm not opposed that idea, I think it would be nice (like Yeoman, Bower, they all have some kind of a mascot). But it would need to be really well executed, like [MailChimp's Freddie](http://mailchimp.com/about/brand-assets/).

**Comment by corysimmons** [comment]
More from Ivan. I love this one.

![](http://i.imgur.com/3A6gqvS.png)

**Comment by corysimmons** [comment]
![](https://s3.amazonaws.com/uploads.hipchat.com/1685/79512/cZPk18pe2qe2Pfl/stylus_logo_opt5-01.png)

This draft is my favorite. Ivan's gonna clean it up tomorrow. It makes Stylus look fun and inviting and by using script it looks like it was written with some sort of stylus (which it probably was). But mostly it makes Stylus look fun (which it is).

**Comment by thefoxis** [comment]
That's definitely the best version so far. It needs some loving though, line widths are wrong at some points. 

I think we should also ask couple other designers to review final draft. I know plenty typography-oriented people. Just to make sure we don't end up redesigning the logo in the near future again.

**Comment by kizu** [comment]
Yep, the last version is the best so far, but I agree that we need someone who is good at typography to review it.

Also, I think we need to add something special to the logo, so it won't be just a written name. Right now it seems a bit similar to the Sass' logo. However, I'm not a designer, so I can't say what could we add to it.

**Comment by thefoxis** [comment]
Having something besides text is what I've initially suggested, although Ivan said that it might be too much with script font. Personally, I think we could combine something very simple (maybe also keep it connected with the font itself).

That's also one of the reasons why I was recommending reaching out to other designers for feedback / submissions.

**Comment by corysimmons** [comment]
@thefoxis I tried to reach out to the community on Twitter and reddit and no one joined in. Would you mind asking some of your designer friends to take a look at it and give us their thoughts on adding a mark to it. TBH, I was thinking about that too, but Ivan might be right about it being redundant to have a mark representing a stylus (or something similar), and have script font (written with a stylus). If we can think of something perfect or very subtle, then that's fine, but I think I'm with Ivan in that I don't want to add junk to it just so we'll "have a mark".

Regardless, here are some ideas.

#### Mark ideas
1. What about adding `()` to the end of it? It represents mixins, functions, etc. and is generally the only required syntax to create functionality in Stylus. I can imagine putting this on the website too with some other script font that animates (slides in/out from left) to expand/contract the parens. Then we could do something like `stylus(is easy)`, `stylus(is fun)`, etc.
2. Could add a stylus of some sort to the left side of the script leaning this way `/`. This might be redundant - I'm just throwing stuff out there.
3. Styluses are also those little [needle-holders](http://www.needledoctor.com/Replacement-Stylus) on old record players (I think). Maybe we could go in that direction a bit and add a record  player? This could open up some design decisions revolving around music.
4. Speaking of branding, maybe we could just do a mascot as mentioned early. Github has Octocat, we could have some type of [fast-ish/fun-ish animal](http://en.wikipedia.org/wiki/Black-tailed_jackrabbit) illustrated mascot.

Just throwing ideas out there. Don't crucify me. :)

**Comment by r77adder** [comment]
Guys, wait a second...
Are we doing a logo for you or for all the friends you are going to ask to approve?
The logo is yours and should represent your thing...not some people expectations, designers or not...
On the other topic of adding one more stylus.... Are you kidding me... the text is saying "stylus", it's written with "stylus"....now you want to add stylus next to it :D 

Further to this, the logo doesn't need to communicate anything even remotely close to what the company/product does. The only purpose of the logo is to be identifier, which is partly carried out by the name. And the logo is only part of the identity, a very small part. My point here is that trying to design the logo to carry all the product identity is a recipe for identity project failure.

Making the logo modest clear and simple and leaving all the other parts in the identity like typefaces, color, layout and weight balance play it's part is the right approach especially in the digital medium, where the logo is always on topleft not more that 100px wide, and it's almost invisible, because it's always there...

I see some sense in adding something like brackets, but with that style I don't see a good way to implement code brackets in a good enough way as I told Cory yesterday...

Sorry for the long post, just a few thoughts... I'm doing it cause I see how this is going in the wrong direction. 
Since I see there is certain doubt about the direction of the design and I there isn't centralised feedback, I'll finish the what I started and you guys are free to use it or not use it.

Cheers,
Ivan

**Comment by corysimmons** [comment]
@r77adder :+1: and I love the new logo. Clean it up a bit and let's roll with it.

**Comment by thefoxis** [comment]
> Are we doing a logo for you or for all the friends you are going to ask to approve?

No. But we want best quality possible. We're not going to reiterate over and over again in the course of months. (I'm not impying that someone is doing non quality work here) Also this is an Open Source project, so thus the design is Open Source. Don't be surprised we will be asking for feedback and opinions. Because that's working in the open. We're forming a team that will collaborate on design. We will ask for feedback. So you need to be comfortable with that.

>  I'm doing it cause I see how this is going in the wrong direction. 

What specifically is going the wrong direction? It's kind of a discouraging statement. I think @corysimmons and @kizu are just throwing ideas around, there's nothing wrong with that. They also have no design background.

The idea of Open Source design is to welcome all submissions, give feedback, pick the best ones that are useful and go along with the project goals. There's no centralised feedback, there are maintainers and collaborators who usually give feedback.

**Comment by corysimmons** [comment]
@thefoxis I agree with your points. I just really don't want this to turn into a several month long endeavor until it loses steam. I know quality > speed, but the logo is good and I'd really like to get this project out there while we have the resources to commit work to it.

I think open source should evolve to become an iterative, fast, and enjoyable experience. I think we all have a tendency to over-analyze/paralyze things and then drag out the feedback until people who were originally interested in being contributors become disenchanted with the whole process.

Here's a perfect example: https://github.com/LearnBoost/stylus/pull/1131

In it, Sascha starts by contributing a lot of useful stuff; tries his hardest to participate in a dialogue to turn it into what everyone wants; everyone genuinely wants different things; this goes on for months; he becomes disenchanted, curses open source, and closes his PR never to work on Stylus again.

This isn't to say the discussion wasn't warranted and didn't eventually end up with a good result, but we definitely ostracized a potential contributor by being slow about it. Decisions need to be made sometimes, and I don't think, in the case of rebranding, that it's particularly threatening to the project's health if we move forward with a certain design that's obviously infinitely better than what came before it, and is high quality regardless of various opinions.

In short, I'm not opposed to critique so long as it comes swiftly and there is some sort of direction that everyone agrees on, but so far as Ivan's concerned, we just don't have the resources to continue having one of our best designers work on iteration after iteration of this logo for weeks to come.

**Comment by thefoxis** [comment]
Yeah, sure, I totally agree the faster the better. I know how it goes :) 

I wasn't trying to say the logo was bad, I like the handwritten, swishy type, it just needs to be finished (as it was mentioned somewhere at the top).

**Comment by corysimmons** [comment]
@thefoxis hehe, I think we're both in the same boat so far as working a full-time job, maintaining open source, and contributing to more open source. So yeah, the sooner we can do this _well_ (see: not perfect) and get it done, the better for everyone.

Ivan is definitely polishing up that last logo and I think we should run with it. I don't mind to get feedback from other designers and see if they want to contribute, but if we don't get anything better by the time we're done with the site, we shouldn't wait.

**Comment by corysimmons** [comment]
![](http://i.imgur.com/prnfCcC.png)

I think this is Ivan's final iteration. He's brushing it up now and will upload the .ai to your LayerVault. Thoughts everyone? @kizu @thefoxis

**Comment by thefoxis** [comment]
:thumbsup: when finalized (just some minor things, such as the thin white line right underneath the crossbar of 't', which I think should be black)

**Comment by corysimmons** [comment]
I think the thin light lines (one between the bottom of the 'y' as well) are on purpose. It seems to give the logo a bit more depth(? it gives the lines a sense that they were drawn in a certain order, like the long line of the 't' was drawn first, then it was crossed), especially at smaller sizes:

![](http://i.imgur.com/ztbCWsU.png)

**Comment by r77adder** [comment]
![screenshot 2014-03-05 21 53 55](https://f.cloud.github.com/assets/6740245/2338880/e72d1d68-a4ae-11e3-8ca7-ad029db2f106.png)

This is called ink trapping... Typeface print technique, those sections are used to absorb the excess of ink in the letter intersections, which otherwise will create a ink blob. As @corysimmons pointed out in the same way they keep the logo crisp in small sizes and provide an extra touch when you see it in large proportions.
I'm not too much of an advocate to those I'm still polishing there.

**Comment by corysimmons** [comment]
![](http://i.imgur.com/iExTdih.png)

Last one. Not sure if I like the arched one or the straight one better.

@thefoxis @kizu @visionmedia chime in and Ivan will finalize one and we can get an .ai to play with.

**Comment by r77adder** [comment]
![screenshot 2014-03-05 23 51 04](https://f.cloud.github.com/assets/6740245/2339669/a684acf4-a4b8-11e3-9fab-3ab8702d7cdf.png)
Here are both just to make it easier for you guys, I'm all in for the straight, just because the curvier one kills the motion of the type at the end part there. I'll work on it a bit but... I still think it won't improve that much...

**Comment by r77adder** [comment]
Here is fixed curvier baseline version...I'm on the fence right now but I'm tipping toward the curved I think
![screenshot 2014-03-06 00 59 18](https://f.cloud.github.com/assets/6740245/2340381/5111e368-a4c2-11e3-90fa-8bc667c19f72.png)
you can also check dribbble http://drbl.in/kqOm

**Comment by corysimmons** [comment]
Damn, love them with color so much. Good work Ivan! =)

**Comment by kizu** [comment]
The latest one is the best, but I still would like someone with a lot of typography experience to look at it. @thefoxis, you said you have some typography-oriented friends, when could you get in touch with them? As Ivan is on fire, it would be nice to provide the feedback soon :)

Also, some questions from me:
1. Can we reduce the whole width of the lines a bit? Not much, but to make it not look that bold?
2. Shouldn't there be a connection between `u`and `s`?
3. I like the curved variant more, but I can see that the connections (between `t` and `y`, and between `l` and `u`) are better in the straight one, is it fixable in the curved?
4. For my (untrained) eye, `y` looks a bit massive now, can we reduce the width of its vertex just a bit?

Anyway, @r77adder, great work!

**Comment by thefoxis** [comment]
+1 on the latest on the right. I also agree with the feedback above re: lettering/kerning.

@kizu I'm going to reach out to people today.

**Comment by corysimmons** [comment]
![](http://i.imgur.com/IPW3kj9.png)

I'm a huge fan of this one. I'm not sure how much more time Ivan has to devote to this (as he's starting work on [Jeet](https://github.com/mojotech/jeet) rebranding), but we'll get the .ai and if anyone wants to take a stab at modifying it while we finish up the redesign/development (before it's public) they certainly can.

TBH, I'm 100% satisfied with it. Thanks Ivan! =)

Love those colors too.

**Comment by thefoxis** [comment]
Yep, thanks! I'll quickly send it to review by fellow typographers.

Colors are no-go in my opinion. We were discussing a light color scheme, and somehow I really don't dig dark schemes for documentation and framework sites. I did a survey on documentation recently and the best documentation websites that were mentioned are all light-colored. Since it's a lot of reading I'd recommend going light.

The green is really strong and borderline offensive to me. I don't like it.

**Comment by corysimmons** [comment]
@thefoxis No problem. I wasn't thinking for the entire site, simply some frame-like sidebar nav, and then have the actual docs be light, but I'll bow to UI designer judgment on this - especially as it seems you've done some research on this exact subject. =)

**Comment by tj** [comment]
I agree with @thefoxis regarding dark schemes. IMO the green is too bright as well but I like the logo overall. Something about it makes it a little difficult to read when it's large for some reason, the letters almost blend a little too well. Maybe site-wise something simple like http://ripplejs.github.io/ would work. I'm a big fan of white hahahha

**Comment by thefoxis** [comment]
@visionmedia &mdash; I'm totally sold on white and a lot of white space. Happy to go towards segment.io direction, they have nice stuff :)

**Comment by kizu** [comment]
Yep, I'm also for the bright themes, although I like it when there is a slight tint of a color or grayness in there, like at http://ia.net/ or [at my site](http://kizu.ru/en/) :)

**Comment by corysimmons** [comment]
Dark themes can be done [well](http://www.unheap.com/) but I agree. I was just thinking _maybe_ (I really don't care that much about it hehe) for a [sidebar](http://mojotech.github.io/jeet/) and then keep all the body area light.

Just throwing ideas out there guys! :)

**Comment by r77adder** [comment]
Guys, The colors are up to you... I just dribbbled it with invert colors :)

I'll be shifting my attention elsewhere, since I think this is a good enough to start working on the UI. Other than that, gather feedback here, I'll see if I can schedule some more time for final touches. 

Per @kizu feedback – Привет, Братушка! :)
1. What's wrong with being bold? – I think reducing the width will flatten the think-thin rhythm. I see that we need to fix some of the column widths of individual letters like L and y but in general I think bolder the better
2. There isn't any rule that says that they should be connected. In cursive script font is achieved by crossing through the width of the letter s to connect with the top bowl. In our case that will reduce the readability. Plus, if we do it for "us" we have to do it for "st" that will make it even harder to read.
3. Better in what way? – the the straight one they are more open, this is something that might need some fine adjustments.
4. For my eye the y is fine the problem is cause by the thinner "L" I think, which probably makes you feel that the problem is in the y which looks massive next to that thin L

Cheers,
Ivan

**Comment by tj** [comment]
@r77adder cool man, nice work! thanks :D

**Comment by thefoxis** [comment]
thanks @r77adder, can you upload the latest source to LayerVault?

good work!

**Comment by r77adder** [comment]
The file is in Layer vault... I've made some extra brand DNA suggestions...
![screenshot 2014-03-06 19 01 37](https://f.cloud.github.com/assets/6740245/2348577/6ada6580-a559-11e3-9cf5-2e451112354a.png)
![screenshot 2014-03-06 18 56 15](https://f.cloud.github.com/assets/6740245/2348578/70df7fd8-a559-11e3-9976-4c42e50d7a6f.png)
![screenshot 2014-03-06 18 56 09](https://f.cloud.github.com/assets/6740245/2348582/76efc464-a559-11e3-9a9c-3f4c4ea03d64.png)

Cheers,
Ivan

**Comment by corysimmons** [comment]
Gorgeous. Thanks

**Comment by thefoxis** [comment]
Thanks.

I think we need to go with a little bit different scheme though, this resembles Sass too much in my opinion.

**Comment by corysimmons** [comment]
Yup, np. I think he was just showing kinda how the logo would look on a few different pages.

**Comment by corysimmons** [comment]
@r77adder I don't think the final version of the logo (the arched one with the underline) is in LayerVault - not so far as I can see anyway.
